VisualStudioDiscordRPC.Shared/PackageController.cs

```diff
@@ -61,6 +61,7 @@ public PackageController(DTE2 instance, string installationPath)
                 Dte = _instance,
                 ExtensionAssets = extensionsAssetMap,
 
+                Enabled = bool.TryParse(Settings.Default.RichTextEnabled, out bool richTexEnabled) && richTexEnabled,
```

Comment:
Here is a typo in richTextEnabled.

Reply:
You are right, it should be richPresenceEnabled